sourcegraph/sourcegraph
# web: remove integrations banners

Closes https://github.com/sourcegraph/sourcegraph/issues/38625
Removes browser and IDE extensions banners from Sourcegraph website (see screenshots in [this comment](https://github.com/sourcegraph/sourcegraph/issues/38625#issuecomment-1182875582)). 

## Test plan
- `sg start web-standalone`
- go to search results page, reload it not less than five times and ensure that neither browser extension nor IDE extensions banner is shown
- go to file (blob) page, hover on different code tokens not less than five times, then hover over View on the code host icon button and ensure that the browser extension popup is not shown (click should take you to the code host)
- tests un CI pass
